src/composables/multisigAccounts.ts

```diff
@@ -49,8 +50,14 @@ export function useMultisigAccounts({ store }: IDefaultComposableOptions) {
 
   const accounts = computed<IAccount[]>(() => store.getters.accounts);
   const activeNetwork = computed<INetwork>(() => store.getters.activeNetwork);
-  const activeMultisigAccount = computed<IMultisigAccount | undefined>(() => multisigAccounts.value
-    .find((account) => account.gaAccountId === activeMultisigAccountId.value));
+  const allMultisigAccounts = computed<IMultisigAccount[]>(
+    () => [...multisigAccounts.value, ...pendingMultisigAccounts.value],
+  );
+
+  const activeMultisigAccount = computed<IMultisigAccount | undefined>(
```

Comment:
activeMultisigAccount can be undefined? if not you can change te type and add on end of "find" "!"

Reply:
yes